lib/simplifyExpression/fractionsSearch/cancelLikeTerms.js

```diff
@@ -257,16 +256,25 @@ function cancelTerms(numerator, denominator) {
     numerator.args[1] = newExponent;
     return new CancelOutStatus(numerator, null, true);
   }
-
   // case 3: they're both polynomial terms, check if they have the same symbol
   // e.g. 4x^2 / 5x^2 => 4 / 5
   // e.g. 4x^3 / 5x^2 => 4x^(3-2) / 5
+  // Case 3.1: they're both polynomial terms with different symbols but with coefficients
+  // e.g 20x / 40y => 1x / 2y
```

Comment:
this would be `x` not `1x` right?
can you add these three examples to the test suite?